projects/cdk/utils/miscellaneous/is-empty.ts

```diff
@@ -0,0 +1,9 @@
+import {tuiIsString} from './is-string';
+
+type EmptyValue = '' | null | undefined;
+
+export function tuiCheckValueIsEmpty<T>(value: EmptyValue | T): value is EmptyValue {
+    const nextValue: any = tuiIsString(value) ? value.trim() : value;
+
+    return [undefined, null, NaN, ``].includes(nextValue);
```

Comment:
That wouldn't work for `NaN`, right?

Reply:
yes, we don't need use NaN in runtime

Reply:
fixed

Reply:
Oh, I didn't know `includes` uses `Object.is` under the hood, I thought it wouldn't work due to `NaN !== NaN`